demos/apps/bookinfo/README.md

```diff
@@ -35,35 +35,46 @@ rename the tool to `istioctl`. For example:
 $ cp istioctl-osx /usr/local/bin/istioctl
 ```
 
-> Note: The following instructions assume your current working directory 
+> Note: The following instructions assume your current working directory
 > is the [istio repo root](https://github.com/istio/istio).
 
 ## Setup
 
-Execute the [Istio installation instructions](../../../kubernetes/README.md) 
+Execute the [Istio installation instructions](../../../kubernetes/README.md)
 to install the Istio manager, mixer, and an envoy-based ingress controller,
 which will be used to implement the gateway for the application.
 (Note: the current version of the bookinfo application MUST use the `default` Kubernetes namespace.)
 
 ## Running the Bookinfo Application
 
 1. Change your current working directory to the bookinfo application directory:
-   
+
    ```bash
    cd demos/apps/bookinfo
    ```
 
 1. Bring up the application containers:
 
    ```bash
-   $ kubectl create -f bookinfo-istio.yaml
+   $ source ../../../kubernetes/istioctl-kube-inject.sh
+   ```
+
+   `istio-inject` is a wrapper function around `istioctl
+   kube-inject` which injects the istio runtime proxy into kubernetes
+   resources files. `istio-inject` and `istioctl kube-inject` are documented [here](https://github.com/istio/istio/blob/master/doc/istioctl.md#kube-inject).
+
+   ```bash
+   $ isito-inject bookinfo-istio.yaml | kubectl apply -f -
```

Comment:
I'm not sure maintaining and using this istio-inject script is worth it. Maybe it would be better to just provide the whole `istioctl kube-inject ...`  command here instead. I think we can hardcode the hub (docker.io/istio) in the command and provide a simple grep/awk command to extract the correct tag from .../.../.../kubernetes/istio-install/istio-manager.yaml.